table/snapshots_test.go

```diff
@@ -0,0 +1,115 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package table_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/apache/iceberg-go/table"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func testSnapshot() table.Snapshot {
+	parentID := int64(19)
+	manifest, schemaid := "s3:/a/b/c.avro", 3
+	return table.Snapshot{
+		SnapshotID:       25,
+		ParentSnapshotID: &parentID,
+		SequenceNumber:   200,
+		TimestampMs:      1602638573590,
+		ManifestList:     manifest,
+		SchemaID:         &schemaid,
+		Summary: &table.Summary{
+			Operation: table.OpAppend,
+		},
+	}
+}
+
+func testSnapshotWithProperties() table.Snapshot {
```

Comment:
do both of these methods need the `test` prefix?

Reply:
not at all, the lowercase `test` prefix was just to indicate they were test snapshots. I can remove the `test` prefix without issue on these.